configmap/Tiltfile

```diff
@@ -39,22 +43,36 @@ def configmap_yaml(name, namespace="", from_file=None, watch=True):
       generator = True
     else:
       fail("Bad from_file argument: %s" % from_file)
+  elif from_env_file:
+    if type(from_env_file) == "list":
+      fail("from_env_file only supports string as an input to prevent confusion with kubectl behavior of only loading the last item in a list")
+    elif type(from_env_file == "string"):
+      args.extend(["--from-env-file", from_env_file])
+      if watch:
+        watch_file(from_env_file)
+      generator = True
 
   if not generator:
     fail("No configmap generator specified")
 
   args.extend(["-o=yaml", "--dry-run=client"])
   return local(args, quiet=True)
 
-def configmap_create(name, namespace="", from_file=None, watch=True):
+def configmap_create(name, namespace="", from_file=None, from_env_file=None, watch=True):
```

Comment:
Same thing here - `from_env_file` will need to be last to keep compatibility for positional arguments.

Reply:
I'm made the requested update, good catch! Was not thinking about the positional implication here!